rpcs3/Emu/Cell/PPUFunction.cpp

```diff
@@ -187,6 +187,7 @@ extern std::string ppu_get_syscall_name(u64 code)
 	case 250: return "sys_spu_thread_group_set_cooperative_victims";
 	case 251: return "sys_spu_thread_group_connect_event_all_threads";
 	case 252: return "sys_spu_thread_group_disconnect_event_all_threads";
+	case 253: return "sys_spu_thread_group...";
```

Comment:
not a good idea to put incomplete names here, since whenever a game use this unimplemented syscall it will get logged as "sys_spu_thread_group..." instead of "syscall_253".  you can see whats the problem with that

Reply:
I saw some incomplete names already inside so I tought it would've been ok to add this.
Should I also remove already existing incomplete names?

Reply:
yup 👌